RDKShell/RDKShell.cpp

```diff
@@ -4358,19 +4358,16 @@ namespace WPEFramework {
 
                     if(!rialtoConnector->initialized())
                     {
-                        string sesEnv,rialtoDebug;
                         LOGWARN("Initializing rialto connector....");
-                        Core::SystemInfo::GetEnvironment(_T("SESSION_SERVER_ENV_VARS"), sesEnv);
-                        Core::SystemInfo::GetEnvironment(_T("RIALTO_DEBUG"), rialtoDebug);
-                        rialtoConnector->initialize(sesEnv,rialtoDebug);
+                        rialtoConnector->initialize();
                     }
                     LOGWARN("Creating app session ....");
                     if(!rialtoConnector->createAppSession(client,display, appId))
                     {
                         response["message"] = "Rialto app session initialisation failed";
                         returnResponse(false);
                     }
-                    if(rialtoConnector->waitForStateChange(appId,RialtoServerStates::INACTIVE,200))
+                    if(rialtoConnector->waitForStateChange(appId,RialtoServerStates::INACTIVE,RIALTO_TIMEOUT_MILLIS))
```

Comment:
The if test is wrong in two places. The correct if test is:

`  if(!rialtoConnector->waitForStateChange(appId,RialtoServerStates::ACTIVE, RIALTO_TIMEOUT_MILLIS))`

See my original patch: https://code.rdkcentral.com/r/c/components/generic/rdk-oe/meta-cmf-video/+/86680/1/recipes-extended/rdkservices/rdkservices/0004-RDKDEV-771-socket-wait-for-active-state.patch#13